evap/results/tests/test_views.py

```diff
@@ -699,3 +701,34 @@ def test_archived_results(self):
         self.app.get(url, user='manager', status=200)
         self.app.get(url, user='reviewer', status=403)
         self.app.get(url, user='student_external', status=403)
+
+
+class TestTextAnswerExportView(WebTest):
+
+    @classmethod
+    def setUpTestData(cls):
+        baker.make(UserProfile, username="student")
+        baker.make(UserProfile, username="reviewer", groups=[Group.objects.get(name="Reviewer")])
+        baker.make(UserProfile, username="manager", groups=[Group.objects.get(name="Manager")])
```

Comment:
student and manager are only used in one test - what do you think about moving their creation in that test method? From my experience, we usually struggle with too much set up being done in the set up methods, not the other way around.